lua/net.lua

```diff
@@ -8,6 +8,12 @@ return function (gatekeeper_server)
 	-- These parameters should likely be initially changed.
 	local log_level = staticlib.c.RTE_LOG_DEBUG
 
+	-- The user that will run Gatekeeper after it boots.
+	local user = "gatekeeper"
+
+	-- The mode that Gatekeeper should set on files after it boots.
+	local mode = gatekeeper.c.S_IRWXU
```

Comment:
`mode` should be the command line parameter `--log-file-mode` like `--log-file-name-format` and `--log-base-dir`. This way, `gatekeeper_log_init()` can call `fchmod(fileno(new_log_file), mode)` after `fopen()` to update the mode.

Reply:
By the way, a good default mode for a log file is `S_IRUSR | S_IWUSR` since one doesn't need the execution right on a log file.

Reply:
Once you are ready, also update the documentation [here](https://github.com/AltraMayor/gatekeeper/wiki/Configuration#gatekeeper-log-files) to add the new parameter `--log-file-mode`.